apps/admin-preset-default/src/components/DefaultToggleInput.vue

```diff
@@ -1,34 +1,41 @@
 <template>
-  <label v-test="[AdminComponent.ToggleInput, element]">
+  <div>
```

Comment:
wrapper divje lijkt me niet persee nodig, of hangt er hier ergens anders nog gedrag/styling aan?